src/lib/containers/Login.tsx

```diff
@@ -146,6 +148,7 @@ class Login extends React.Component<Props, State> {
       })
   }
   public render() {
+    const registerAccountUrl = this.props.registerAccountUrl ?? `${getEndpoint(BackendDestinationEnum.PORTAL_ENDPOINT,)}#!RegisterAccount:0`
```

Comment:
is trailing comma a convention? I saw it in the code that this replaces. Just curious

Reply:
Not sure why that comma was there, but it was a copy/paste error (so I fixed it here, and at the source)